playbooks/roles/ec2-security-group/tasks/main.yml

```diff
@@ -33,6 +33,12 @@
         from_port: "{{ ssh_port }}"
         to_port: "{{ ssh_port }}"
         cidr_ip: 0.0.0.0/0
+      # Let's Encrypt
+      # ---
+      - proto: tcp
+        from_port: "{{ le_port }}"
+        to_port: "{{ le_port }}"
+        cidr_ip: "{{ le_cidr }}"
```

Comment:
I think this can be `0.0.0.0/0` in all cases no?

Reply:
I think you can remove the `le_cidr` var and use that range directly like the other rules.